core/src/main/java/org/apache/iceberg/hadoop/HadoopTableOperations.java

```diff
@@ -53,13 +54,15 @@
 import org.slf4j.LoggerFactory;
 
 /**
- * TableOperations implementation for file systems that support atomic rename.
+ * TableOperations implementation for file systems that support atomic rename. It should only be
```

Comment:
I'd just leave this as is, posix compliance isn't strictly required just the atomic rename